packages/@lwc/synthetic-shadow/src/faux-shadow/element.ts

```diff
@@ -368,6 +377,12 @@ export function PatchedElement(elm: HTMLElement): HTMLElementConstructor {
             // https://github.com/Microsoft/TypeScript/issues/338
             return superBlur.call(this);
         }
+        focus(this: HTMLElement) {
+            disableKeyboardFocusNavigationRoutines();
```

Comment:
what should happen when you call `<custom-element>.focus()` on a component that is delegating focus, has focusable elements, but not a `@api focus()` method defined?